Core/RoverControllerPresets/DriveControllers/GoodOldGamesLikeController.cs

```diff
@@ -6,31 +6,24 @@ namespace RoverControlApp.Core.RoverControllerPresets.DriveControllers;
 
 public class GoodOldGamesLikeController : IRoverDriveController
 {
+	public KinematicMode Mode { get; set; } = KinematicMode.Compatibility;
+
 	public RoverControl CalculateMoveVector()
 	{
 		//deadzone have to be non zero for IsEqualApprox
 		var joyDeadZone = Mathf.Max(
 			0.001f,
 			Convert.ToSingle(LocalSettings.Singleton.Joystick.Deadzone)
 		);
-
-		Vector2 velocity = Input.GetVector(
-			"rover_move_down",
-			"rover_move_up",
-			"rover_move_right",
-			"rover_move_left",
-			joyDeadZone
-		);
+		Vector2 tempVel = Input.GetVector("rover_move_down", "rover_move_up", "rover_move_right", "rover_move_left", joyDeadZone);
+		Vector3 velocity = new Vector3(tempVel.X, tempVel.Y, 0);
 
 		if (LocalSettings.Singleton.SpeedLimiter.Enabled)
 			velocity *= LocalSettings.Singleton.SpeedLimiter.MaxSpeed;
 
 		velocity.X = Mathf.IsEqualApprox(velocity.X, 0f, joyDeadZone) ? 0 : velocity.X;
 		velocity.Y = Mathf.IsEqualApprox(velocity.Y, 0f, joyDeadZone) ? 0 : velocity.Y;
 
-		if (Input.IsActionPressed("camera_zoom_mod"))
-			velocity /= 8f;
-
-		return RoverControlVec2Extension.FromVector2(velocity);
+		return RoverControlVec3Extension.ToRoverControl(velocity);
```

Comment:
Prefer using extension methods from extended classes.
```suggestion
		return velocity.ToRoverControl();
```
see:
https://github.com/wisniax/RoverControlApp/blob/3b3e3cfab920852433be6b94e847b16dd4c70398/Core/RoverControllerPresets/MqttClasses.RoverControlVec3Extension.cs#L12-L13

Reply:
Adressed in [5ed280b](https://github.com/wisniax/RoverControlApp/pull/36/commits/5ed280b9da4555ee40b428e3d684a0ecffc34549)